absl/flags/flag.h

```diff
@@ -155,7 +155,7 @@ void SetFlag(absl::Flag<T>* flag, const V& v) {
 //
 // where:
 //
-//   * `T` is a supported flag type (See below),
+//   * `T` is a supported flag type (see `absl/flags/marshalling.h`),
```

Comment:
For consistency, we tend to reference just the file name when we're staying within the current directory. 
```suggestion
//   * `T` is a supported flag type (see `marshalling.h`),
```